kernelci/build.py

```diff
@@ -1107,6 +1119,8 @@ def run(self, jopt=None, verbose=False, opts=None):
             env['name'],
         ])
 
+        publish_path = publish_path.replace(':', '-')
```

Comment:
This is a bit hacky but let's see if it works and then we can refine the implementation.

Reply:
ok, works for me